### ISSUE TITLE
[ENHANCEMENT] The nav links are not visible in dark mode due to bad color contrast

### ISSUE THREAD

**Issue by StephanosNikitis** [Nikhilrsingh/car-transport-service#673]
## ⚙️ Current Behavior

Nav links are not properly visible in dark mode due to bad color contrast.

<img width="1746" height="98" alt="Image" src="https://github.com/user-attachments/assets/7a018bb4-5c95-428f-a5a1-97140444888b" />

---

## 🚀 Proposed Improvement

Fix the CSS so that nav links can be visible in dark mode.

---

## 🔍 Why It’s Needed

It is needed for better UI/UX.

---

## 🧩 Possible Implementation

Fix the CSS of nav links so that they are clearly visible during dark mode also.

---

## ✅ Checklist

- [☑️] I’ve reviewed existing issues to ensure this isn’t a duplicate.  
- [☑️] I’ve explained how this improves performance or readability.  

Please assign me this issue under SWOC.
